request.go

```diff
@@ -107,7 +107,7 @@ func (req *Request) convertOptions(root *Command) error {
 		}
 
 		kind := reflect.TypeOf(v).Kind()
-		if kind != opt.Type() {
```

Comment:
Can we add a comment here on why we don't need to convert string options?

Reply:
I reverted the change that excluded options with  `Type==Strings`. I used an older PR as a starting point that had that exception, but I noticed that it would allow requests constructed with `NewRequest(...)` set non-array values. Would that comment still be necessary?

My understanding is that options with `Type==String==reflect.String` is not needed because `v` is already a string and does not need to be converted.